files/vscode/extensions.txt

```diff
@@ -0,0 +1,24 @@
+AndrsDC.base16-themes
+axiomaticstudios.one-monokai-80s
+bung87.rails
+bung87.vscode-gemfile
+castwide.solargraph
+christian-kohler.npm-intellisense
+christian-kohler.path-intellisense
+dbaeumer.vscode-eslint
+eamodio.gitlens
+eg2.vscode-npm-script
+jasonnutter.search-node-modules
+jawandarajbir.react-vscode-extension-pack
+Lourenci.go-to-spec
+mgmcdermott.vscode-language-babel
+misogi.ruby-rubocop
+ms-vscode.atom-keybindings
+ms-vscode.sublime-keybindings
+msjsdiag.debugger-for-chrome
+noku.rails-run-spec-vscode
+sianglim.slim
+vayan.haml
+xabikos.JasmineSnippets
+xabikos.JavaScriptSnippets
+xabikos.ReactSnippets
```

Comment:
here's a few suggestions

```
waderyan.gitblame
rebornix.ruby
tyriar.sort-lines
```

Reply:
The git extension we have now is a bit more interactive and detailed, so the `gitblame` extension may be redundant. The other two are great!

Reply:
ah, and one more
`vscode-icons-team.vscode-icons`